src/test/emulators/storage/metadata.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { expect } from "chai";
+import { toSerializedDate } from "../../../emulator/storage/metadata";
+
+describe("toSerializedDate", () => {
+  it("correctly serializes date", () => {
+    const testDate = new Date("2022-01-01T00:00:00.000Z");
```

Comment:
can you init this date with a custom timezone to verify it converts to UTC properly?